config/routes.rb

```diff
@@ -15,6 +15,7 @@
 
   scope 'tournaments', controller: 'tournaments' do
     get ':id' => :show, as: :tournaments_show
+    get ':id/pools' => :show_pools, as: :show_pools
```

Comment:
I'd say the path should be `tournament_pools` rather than `show_pools`. In Rails, `show` is usually reserved for the seven common `resources` for each controller.